spec/models/driver_spec.rb

```diff
@@ -9,19 +9,22 @@
   describe 'driver validations' do
     subject { driver.valid? }
 
-    let(:driver) { Driver.new(
-      name: name,
-      license_number: license_number,
-      mobile_number: mobile_number,
-      email_address: email_address,
-      age: age,
-      status: status,
-    ) }
+    # let(:driver) { Driver.new(
+    #   name: name,
+    #   license_number: license_number,
+    #   mobile_number: mobile_number,
+    #   email_address: email_address,
+    #   age: age,
+    #   status: status,
+    # ) }
+    let(:driver) do
+      build(:driver, name: name, license_number: license_number, mobile_number: mobile_number, email_address: email_address, age: age, status: status)
```

Comment:
line too long, lets space it out like line:13-18 and then get rid of that commented out code